core/src/wsv.rs

```diff
@@ -238,13 +238,34 @@ impl<W: WorldTrait> WorldStateView<W> {
             .drain(..)
             .collect();
 
-        let triggers = self.world.triggers.find_matching(
-            events
-                .iter()
-                .chain(&block.as_v1().event_recommendations)
-                .chain(once(&time_event)),
-        );
-        self.execute_triggers(triggers).await?;
+        let actions = Arc::new(RwLock::new(Vec::new()));
+        self.world
+            .triggers
+            .inspect_matching(
+                events
+                    .iter()
+                    .chain(&block.as_v1().event_recommendations)
+                    .chain(once(&time_event)),
+                |action| -> Result<()> {
+                    // Can't just use the next line to process actions cause of `tracing` crate bug.
+                    // Either use temporary vector `actions` (like now), either remove `#[log]`
+                    // for every `Execute` implemetor inside `isi/mod.rs`
+                    //
+                    // self.process_executable(&action.executable, &action.technical_account)
```

Comment:
I'd prefer to remove `log` macro from `isi/mod.rs` but decided not to do it for now. What do you think?